fr.free.Qccrypt.yml

```diff
@@ -0,0 +1,53 @@
+app-id: fr.free.Qccrypt
+runtime: org.kde.Platform
+runtime-version: 5.15-22.08
+sdk: org.kde.Sdk
+command: qccrypt
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --filesystem=host
+  - --device=dri
+modules:
+  - name: qccrypt
+    buildsystem: simple
+    build-commands:
+      - install -D qccrypt /app/bin/qccrypt
+      - install -Dm644 ${FLATPAK_ID}.desktop -t /app/share/applications
+      - install -Dm644 qccrypt-16-x16.png /app/share/icons/hicolor/16x16/apps/${FLATPAK_ID}.png
+      - install -Dm644 qccrypt-24-x24.png /app/share/icons/hicolor/24x24/apps/${FLATPAK_ID}.png
+      - install -Dm644 qccrypt-32-x32.png /app/share/icons/hicolor/32x32/apps/${FLATPAK_ID}.png
+      - install -Dm644 qccrypt-48-x48.png /app/share/icons/hicolor/48x48/apps/${FLATPAK_ID}.png
+      - install -Dm644 qccrypt-64-x64.png /app/share/icons/hicolor/64x64/apps/${FLATPAK_ID}.png
+      - install -Dm644 qccrypt-96-x96.png /app/share/icons/hicolor/96x96/apps/${FLATPAK_ID}.png
+      - install -Dm644 qccrypt-128-x128.png /app/share/icons/hicolor/128x128/apps/${FLATPAK_ID}.png
+      - install -Dm644 qccrypt-256-x256.png /app/share/icons/hicolor/256x256/apps/${FLATPAK_ID}.png
+    sources:
+      - type: file
+        path: qccrypt
+      - type: file
+        path: fr.free.Qccrypt.desktop
+      - type: file
+        path: qccrypt-16-x16.png
+      - type: file
+        path: qccrypt-24-x24.png
+      - type: file
+        path: qccrypt-32-x32.png
+      - type: file
+        path: qccrypt-48-x48.png
+      - type: file
+        path: qccrypt-64-x64.png
+      - type: file
+        path: qccrypt-96-x96.png
+      - type: file
+        path: qccrypt-128-x128.png
```

Comment:
all these icon size are unecessary. only 256x256 is useful here.

Reply:
Thank you for your help. I will remove them and keep only 256x256.